core/drivers/corda-driver/makefile

```diff
@@ -2,6 +2,7 @@ DOCKER_IMAGE_NAME ?= weaver-corda-driver
 DOCKER_TAG ?= 1.2.3
 DOCKER_REGISTRY ?= ghcr.io/hyperledger-labs
```

Comment:
Same refactoring of registry and image names needed here? Also, see my earlier comment on this.